python/graphstorm/model/embed.py

```diff
@@ -208,17 +209,14 @@ def __init__(self,
 
         self.activation = activation
 
-        # NCCL backend is not supported for utilizing learnable embeddings on nodes. It has a
-        # dependency on distDGL (PR https://github.com/dmlc/dgl/pull/5929).
-        # TODO (Israt): Add NCCL support to distDGL.
-        if is_distributed() and get_backend() == "nccl":
+        if dgl.__version__ <= "1.1.2" and is_distributed() and get_backend() == "nccl":
             if self.use_node_embeddings:
-                raise NotImplementedError('NCCL backend is not supported for utilizing \
-                    node embeddings. Please use gloo backend.')
+                raise NotImplementedError('NCCL backend is not supported for utilizing ' +
```

Comment:
can you also mention that people can potentially use a newer version of DGL?

Reply:
Added in the Error comment. Does additional comments required?